_run/kube/deployment.yaml

```diff
@@ -6,22 +6,14 @@ services:
     image: quay.io/ovrclk/demo-app
     expose:
       - port: 80
-        as: 80
```

Comment:
I kept this diff to show example of deployment i used to create NodePort for randpom port assigments